lib/routes/read.js

```diff
@@ -5,20 +5,23 @@
 import type {RouteConfiguration} from '../../types.js'
 */
 
-const projectMeta = require('../utils/project-meta.js')
+const scope = require('../scope.js')
 const project = require('../project.js')
+const values = require('../values.js')
 
 function readRoutes (
   cwd /* : string */
 ) /* : Promise<Array<RouteConfiguration>> */ {
-  return projectMeta.read(cwd)
+  return scope.read(cwd)
     .then((config) => {
-      if (config.server && config.server.routes) {
-        return config.server.routes
-      }
-      return project.listRoutes(cwd)
+      return Promise.resolve()
+        .then(() => config.routes || project.listRoutes(cwd))
+        .then((routes) => routes || [])
+        .then((routes) => routes.map((route) => {
```

Comment:
True.

I have a habit of making my `.then()`s do only one thing, which prob is not great for performance. Ill keep this in mind for the future.